src/components/ChecWysiwyg.vue

```diff
@@ -0,0 +1,571 @@
+<template>
+  <div class="wysiwyg" :class="classNames">
```

Comment:
```suggestion
  <div class="chec-wysiwyg" :class="classNames">
```

We've have conflicts with generic class names like this before